app/config/config.yml

```diff
@@ -20,6 +20,7 @@ parameters:
 framework:
     #esi:             ~
     #translator:      { fallbacks: ["%locale%"] }
+    csrf_protection: ~
```

Comment:
I thought symfony forms had default CSRF protection. Good that we are implementing now

Reply:
I'm not sure how well it's implemented, but I had to add a hidden field to this form in html myself.